elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -141,7 +142,10 @@ def process(self, pipeline):
             )
 
     def export(self, pipeline, pipeline_export_format, pipeline_export_path, overwrite):
-        if pipeline_export_format not in ["py"]:
+        export_file_types = [
+            file_type['id'] for file_type in RuntimeTypeResources.get_instance_by_type(self._type).export_file_types
+        ]
```

Comment:
@kevin-bates thoughts on this being overkill? (also applies to `processor_kfp`)

Reply:
I don't understand what this statement is doing - at least at first (and second glance). :smile:

After 10 minutes I think I see what this is doing since the export types are dictionaries.  If the idea is to gather up the export file types (extensions), I think something like this is necessary.  Of course, I'm always in favor of supportability and readability.  

Perhaps just adding a comment or two, or burying this in a method  like `get_export_extensions()` on `RuntimeTypeResources` which performs this action, then the line of code could be:

```python
export_file_types = RuntimeTypeResources.get_instance_by_type(self._type).get_export_extensions()
```
and the method would look something like:
```python
def get_export_extensions(self) -> List[str]:
       return [
            file_type['id'] for file_type in self.export_file_types
        ]
```

Reply:
Good idea -- I like the idea of creating a new method, and I'll add some comments/docstrings!